regression/goto-harness/nondet_strings/test.desc

```diff
@@ -0,0 +1,8 @@
+CORE
+main.c
+--harness-type call-function --function function --treat-pointer-as-cstring pointer --associated-array-size pointer:size
+\[function.assertion.\d+\] line \d+ assertion pointer\[size - 1\] == \'\\0\': SUCCESS
+^VERIFICATION SUCCESSFUL$
+^EXIT=0$
+^SIGNAL=0$
+--
```

Comment:
It would be good to also have a test with an assertion which should fail.

Reply:
Seems reasonable, I'll write one.

Reply:
This is now done, thanks for the suggestion.